whylog/teacher/constraint_links_base.py

```diff
@@ -33,27 +33,28 @@ def add_links(self, links):
         self.links = list(set(self.links).union(constraint_links))
 
     def remove_links_by_line(self, line_id):
-        self._remove_links(self._select_by_line(line_id))
+        return self._remove_links(self._select_by_line(line_id))
 
     def remove_links_by_group(self, line_id, group_no):
-        self._remove_links(self._select_by_group(line_id, group_no))
+        return self._remove_links(self._select_by_group(line_id, group_no))
```

Comment:
What do you mean? I decided to use line_id and group_no everywhere instead of object group.